_includes/parse-server/upgrading-to-v3.md

```diff
@@ -0,0 +1,245 @@
+# Upgrading Parse Server to version 3.0.0
+
+Starting 3.0.0, parse-server uses the JS SDK version 2.0.
+In short, parse SDK v2.0 removes the backbone style callbacks as well as the Parse.Promise object in favor of native promises.
+All the Cloud Code interfaces also have been updated to reflect those changes, and all backbone style response objects are removed and replaced by Promise style resolution.
+
+With the 3.0.0, parse-server has completely revamped it's cloud code interface. Gone are the backbone style calls, welcome promises and async/await.
+
+In order to leverage those new nodejs features, you'll need to run at least node 8.10. We've dropped the support of node 6 a few months ago, so if you haven't made the change yet, now would be a really good time to take the plunge.
+
```

Comment:
Most of the content is transferred from [the GitHub README](https://github.com/parse-community/parse-server#upgrading-to-300).

I didn't want any knowledge gaps arising from me cutting out sections.

Would you like me to remove this section? 😊

Reply:
I've gone back and forth on this one but certain we should remove now - we want to avoid the duplication of having the contents of the migration guide here. We could remove the guide on GH but considering the number of links to it I don't think that would be helpful.

Let's just include the same text from the readme with a link to the guide...

> Starting 3.0.0, parse-server uses the JS SDK version 2.0. In short, parse SDK v2.0 removes the backbone style callbacks as well as the Parse.Promise object in favor of native promises. All the Cloud Code interfaces also have been updated to reflect those changes, and all backbone style response objects are removed and replaced by Promise style resolution.
> 
> We have written up a [migration guide](https://github.com/parse-community/parse-server/blob/master/3.0.0.md), hoping this will help you transition to the next major release.

A recent post on the readme of someone who hasn't touched their server since migrating made me realise it should have a mention.

Reply:
I just made the change btw

Reply:
All good!